### ISSUE TITLE
A faster substitute to NSHTMLTextDocumentType?

### ISSUE THREAD

**Issue by Codeido** [iwasrobbed/Down#23]
Hi! 

Can be Down a better and faster substitute to render html inside a textview?
For example, is possible to use Down to improve the rendering of complex html inside a table view cell?
Actually i'm using an attributed string with this option `[NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType]` but the rendering is too slow.

Thanks
-Paolo

**Comment by iwasrobbed** [comment]
@Codeido I haven't tried that before, but I Down is mostly for translating Common Markdown into other types (such as HTML). You'd still need something to render those converted types and Down has no control over that. 

DownView will render HTML since it's a simple web view subclass, but it's still bound by the same rendering performance constraints as any other web view. I would assume UITextView uses a different way of rendering HTML.

tl;dr: I'm not sure, but you're rendering from markdown to html, then yes Down can help speed that up. Down will not help speed up HTML -> View rendering

**Comment by iwasrobbed** [comment]
Please look into the new AST functionality which will performantly render attributed strings in customized ways: https://github.com/iwasrobbed/Down/pull/132
